application/features/mfa.feature

```diff
@@ -138,13 +136,12 @@ Feature: MFA
       | employee_id | 123          |
       | label       |              |
     When I update the mfaWebauthn
-    Then the response status code should be 400
-      And the response body should contain '"label" must have a value in calls to update-webauthn'
+    Then the response status code should be 500
```

Comment:
I think this response status code should probably be 400, since it was a bad request (to update the mfa_webauthn record but providing no label).

Reply:
This came from moving the verification to the `rules` on the model.  What's the best way to do this?

Reply:
I added a later comment suggesting a way to do this just now. :-)